### PR TITLE
feat(content): Clean up descriptions of new spaceport missions

### PR DESCRIPTION
**Pull request by Anarchist2**
The missions were missing payment and some cargo information.

### CODE REVIEW

File: data/human/human missions.txt [hunk 1]
```diff
@@ -6425,7 +6425,7 @@ mission "Care Package to South 2b"
 mission "Moving House 1"
 	minor
 	name "Moving House"
-	description "An anarchist named Kris has asked you to help them move house from <planet stopovers> to <destination>."
+	description "An anarchist named Kris has asked you to help them and <cargo> move house from <planet stopovers> to <destination>. Payment is <payment>."
```
Comment (EjoThims) [inline data/human/human missions.txt:6428]:
```suggestion
	description "An anarchist named Kris has asked you to help them move house with <cargo> from <planet stopovers> to <destination>. Payment is <payment>."
```

The whole "move house" phrasing is awkward to me, but I think this way around is less so?

File: data/human/human missions.txt [hunk 2]
```diff
@@ -6509,9 +6509,9 @@ event "kris gets frustrated"
 
 mission "Moving House 2"
 	minor
-	description "Kris is unsatisfied on <origin> and wants your help to move to <destination> instead."
+	description "Kris is unsatisfied with life on on <destination> and wants your help to move."
```
Comment (EjoThims) [inline data/human/human missions.txt:6512]:
```suggestion
	description "Kris is unsatisfied with life on <destination> and wants your help to move."
```

File: data/human/human missions.txt [hunk 3]
```diff
@@ -6509,9 +6509,9 @@ event "kris gets frustrated"
...
 	landing
-	name "Moving House, Again"
+	name "Moving House, again"
```
Comment (EjoThims) [inline data/human/human missions.txt:6514]: If this is a title format, it should all be capitalized. If not, "House" should be "house."

File: data/human/human missions.txt [hunk 4]
```diff
@@ -6529,7 +6529,8 @@ mission "Moving House 2"
 			`	It takes much less time to load everything this time than the last time you helped Kris move, and soon they hop in the driver's seat of the van, calling to you, "I'll meet you at the spaceport in an hour or two!"`
 
 mission "Moving House 3"
-	name "Moving House, Again"
+	name "Moving House, again"
```
Comment (EjoThims) [inline data/human/human missions.txt:6532]: Same as above.

**Review by EjoThims — CHANGES_REQUESTED** [review]

**Review by EjoThims — APPROVED** [review]

**Review by lumbar527 — APPROVED** [review]

**Pull request merged**